dts/msm8916/msm8916-samsung-r04.dts

```diff
@@ -65,4 +65,15 @@
 			i2c-address = <0x25>;
 		};
 	};
+
+	grandmaxlteub {
+		model = "Samsung Galaxy Max (SM-G720AX)";
+		compatible = "samsung,grandmaxlteub", "qcom,msm8916", "lk2nd,device";
```

Comment:
Where does the `ub` come from?

Reply:
`ro.product.name=grandmaxlteub` in `build.prop`